templates/python/events/func.py

```diff
@@ -0,0 +1,17 @@
+from cloudevents.http import CloudEvent, to_binary
+
+# ---------------------------------------
+# Function template
+# context is a dictionary with the Flask request object and 
+# and a cloud_event key for incoming events
+# ---------------------------------------
+def main(context):
```

Comment:
Just idea:
What about using type hints?
```python3
from werkzeug import Request
from typing import TypedDict


class Context(TypedDict):
    request: Request


def main(context: Context):
    """
    Doc here
    :param context: 
    :return: 
    """
    return {"message": "Howdy!"}, 200
```

Reply:
also I believe that docstring for a function should be written inside `""" """` not using `#`.

Reply:
To be clear: I am not strong proponent of typehints, but they may be useful for APIs.

Reply:
Done. Thanks for the suggestions.